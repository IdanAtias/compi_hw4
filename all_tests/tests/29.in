F x = 1 F;
bool b = x > 1 F;
while ( true and b){
	F z = 10 F;
	while( false or b){
		F y;
		input y;
		if ( y >= x and y <= z){
			print "impossible" ;		
		}
	}
}
C y = 100 C;
%num = 10; 
