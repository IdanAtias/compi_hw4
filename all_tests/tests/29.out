-- end scope --
-- end scope --
var:y:3:FAHRENHEIT
-- end scope --
var:z:2:FAHRENHEIT
line 14: lexical error
